boards/arm/cy8ckit_062_pioneer/cy8ckit_062_pioneer_m4.dts

```diff
@@ -0,0 +1,55 @@
+//SPDX-License-Identifier: Apache-2.0
+//Copyright (c) 2023 David Ullmann
+
+/dts-v1/;
+#include <infineon/psoc6/mpns/CY8C6244LQI_S4D92.dtsi>
+
+/ {
+	model = "Cypress PSoC6 Pioneer Kit";
```

Comment:
```suggestion
	model = "Infineon PSoC 62S4 Pioneer Kit";
```
When written in text, there is always a space between PSoC and any numerical designation. Also, there are lots of PSoC 6 Pioneer Kits, so adding the full name helps set expectations.